README.md

```diff
@@ -74,6 +74,9 @@ You can access these pages on your computer using one of the following clients:
   `pub global activate tldr`
 - [Dash docset](https://github.com/Moddus/tldr-python-dash-docset):
   Open `Preferences > Downloads > User Contributed` and find `tldr pages` in the list
+- Docker images:
+    - [tldr-docker](https://github.com/nutellinoit/tldr-docker) run command via docker container
```

Comment:
Perhaps this would read better as something like


```
- [tldr-docker](https://github.com/nutellinoit/tldr-docker)- Run the `tldr` command via a docker container: `alias tldr='docker run --rm -it -v ~/.tldr/:/root/.tldr/ nutellinoit/tldr'`
```